frontend/src/features/workflow/RunStateIcon.tsx

```diff
@@ -5,20 +5,21 @@ import { RunState } from '../../qrimatic/run'
 
 export interface RunStateIconProps {
   state: RunState
+  size?: "sm" | "xs" | "md" | "lg" | undefined
```

Comment:
no need for the `undefined` in the enumeration if the field is optional
```suggestion
  size?: "sm" | "xs" | "md" | "lg"
```